sunpy/timeseries/timeseries_factory.py

```diff
@@ -46,11 +47,10 @@ class TimeSeriesFactory(BasicRegistrationFactory):
 
     Parameters
     ----------
-    \\*inputs
-        Inputs to parse for timeseries objects. See the example section for a
-        detailed list of possible inputs.
+    inputs : esp, eve, gbmsummary, lyra, noaaindices, noaapredictindices, norh, rhessi, xrs
+        These are the keywords for the source packages currently supported by TimeSeries.
 
-    source : `str`, optional
+    source : {values}
```

Comment:
We should make sure these look like: 

```
order : {'C', 'F', 'A'}
    Description of `order`.
```
from https://numpydoc.readthedocs.io/en/latest/format.html